source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
+author: jbrooks
+date: 2016-02-29 14:00:00 UTC
+tags: ovirt, gluster, centos
+published: false
```

Comment:
Not sure what the "published" flag means here, we didn't use it in any other posts. I guess in this blog's case, if the PR isn't merged then the blog post won't show up anyway.